app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/ssoUsersForm/ssoUsersForm.jsx

```diff
@@ -65,8 +82,27 @@ const SsoUsersFormComponent = ({ enabled: enabledFromStore, fetchAppInfo }) => {
   const getDescription = () =>
     formatMessage(enabled ? messages.ssoOnlyDescription : messages.manualInvitesDescription);
 
-  const handleToggle = (value) => {
-    setEnabled(value);
+  const handleToggle = async (value) => {
+    try {
+      await fetch('/api/v1/settings', {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: token,
```

Comment:
These headers are set by default if the `fetch` utility used, so pls remove it.

Reply:
Done